lib/gringotts/gateways/trexle.ex

```diff
@@ -340,6 +339,13 @@ defmodule Gringotts.Gateways.Trexle do
     }
   end
 
+  defp respond({:ok, %{status_code: code, body: _body}}) when code in [401] do
+    {
+      :error,
+      %Response{reason: "Unauthorized access.", message: "Unauthorized access"}
+    }
+  end
+
   defp respond({:ok, %{status_code: status_code, body: body}}) do
     {:ok, results} = decode(body)
```

Comment:
So what does trexle send in the response body for 401? Is there some JSON?

Reply:
This is the response: